spotify.ts

```diff
@@ -24,22 +26,23 @@ let client: Client;
 
 function setItem(k, v) {
   try {
-    localStorage.setItem(k, v);
+    // localStorage.setItem(k, v);
   } catch {
     // pass
   }
 }
 
 function getItem(k) {
   try {
-    return localStorage.getItem(k);
+    // return localStorage.getItem(k);
   } catch {
     return null;
   }
 }
```

Comment:
I don't think this is needed. The wrappers here ensure the app continues to work without localStorage.

Reply:
The compiler doesn't allow for the localStorage API at all, it crashes if it see it

Reply:
Oh, how can I reproduce it?

It works for me from `main`:
```
deno task compile
./music-app
```

Reply:
Oh, I can't re-create the issue anymore...hmmm...🤔

Reply:
I've uncommented the places localStorage is used